src/systems/systems/src/simulation/mod.rs

```diff
@@ -591,6 +591,12 @@ impl<T: Writer> Write<usize> for T {
     }
 }
 
+impl<T: Reader> Read<usize> for T {
+    fn read(&mut self, name: &str) -> usize {
+        self.read_f64(name) as usize
+    }
+}
```

Comment:
**note**: It is quite likely #5598 will merge into master earlier than this PR. When that happens, you can remove this code as an implementation is already provided: https://github.com/flybywiresim/a32nx/pull/5598/files#diff-59f7c495af8b955154c7a0b33b4322fd79efa182e130bebc23a8046a577af22fR602

Reply:
Removed when resolving merge conflicts 😃.